runtime-config-linux.md

```diff
@@ -130,6 +130,13 @@ Next parameters can be specified:
 Also known as cgroups, they are used to restrict resource usage for a container and handle
 device access.  cgroups provide controls to restrict cpu, memory, IO, and network for
 the container. For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
+The path to the cgroups can be specified in the Spec. The path is expected to be relative
+to the cgroups mount point. The Spec does not support split hierarchy. The cgroups will be 
+created if they don't exist. 
+
+```json
+   "cgroupsPath": "/myRuntime/myContainer"
```

Comment:
I'd mirror [cgexec(1)][1] with something like:

    "cgroups": {
      "join": [
        {
          "path": "/myRuntime/test1",
          "controllers": ["cpu", "memory"]
        },
        {
          "path": "/myRuntime/test2",
          "controllers": ["swap"]
        }
      ]
    }

And it would be nice to move the current "resources" into:

    "cgroups": {
      "create": …,
    }

[1]: http://linux.die.net/man/1/cgexec